integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -512,6 +512,10 @@ public static boolean deletePod(String name, String namespace) {
    */
   public static V1Pod getPod(String namespace, String labelSelector, String podName) throws ApiException {
     V1PodList pods = listPods(namespace, labelSelector);
+    // log pod names for debugging
```

Comment:
Is this debug still needed ifyou plan to check the operator logs?

Reply:
Yes, there are other tests which check the introspector logs and introspector pod status.  If they fail, it will be useful to debug. I have only modified the two failed tests to check the operator log.

Reply:
ok. Can you add more details to the debug log message so it's easy to search and folks would know that it's for debugging?
For e.g. getLogger().info("DEBUG: Pod Name returned by listPods call {0}", pod.getMetadata().getName());

Reply:
done